### DISCUSSION TITLE
Welcome to RemindMe Discussions!

### DISCUSSION THREAD

**Discussion by Stefangansevles** [Stefangansevles/RemindMe#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

  To get started, comment below!

**Comment by hviniciusg** [comment]
Hey, how do you uninstall remind me?

**Reply by Stefangansevles** [reply]
Via the windows Control panel -> Uninstall a program -> search for RemindMe

**Reply by AHGFL0071** [reply]
I'm getting a continuous error message just after downloading it for the first time, that will not allow the program to open correctly.  The error log started on 1/29/2023 and the latest one is 3/09/2023. Just what you didn't want to see. I'm using Windows 10 Pro 64 bit, and have 7.75 TB of hard drive space.
Those mentioned error messages follow [with the spelling errors]:  
[1/29/2023 12:00:58 PM] - Failed to anonymize input 
System.ArgumentException: The path is not of a legal form.
   at System.IO.Path.NormalizePath(String path, Boolean fullCheck, Int32 maxPathLength, Boolean expandShortPaths)
   at System.IO.Path.GetPathRoot(String path)
   at Business_Logic_Layer.BLIO.AnonymizePath(String input)

[3/9/2023 6:10:25 AM] - Loading theme Dark-Red with ID 1 failed
System.ArgumentOutOfRangeException: InvalidArgument=Value of '113' is not valid for 'SelectedIndex'.
Parameter name: SelectedIndex
   at System.Windows.Forms.ComboBox.set_SelectedIndex(Int32 value)
   at RemindMe.MUCTheme.LoadTheme(Themes theme)

I had to force the program to open by using my Administrator level privilege, which I normally would not have to do had the program run as intended.

**Comment by jenl2881** [comment]
Would you consider adding a feature to set up a recurring notification? I know a lot of people have posted online looking for a way to get a notification every hour to keep track of time. Not a reminder of anything specific to do or a timer for 1 hour, but just something to remind me each time the hour changes (1 pm, 2 pm, 3 pm, etc.). I have ADHD and would love a feature like this.

**Reply by Stefangansevles** [reply]
Sorry for the late response, but you could make a reminder and set it's repeat time to hourly, then it will pop up every hour indefinitely @jenl2881 

see: 
![image](https://user-images.githubusercontent.com/25766371/201340467-050ac622-54c1-497d-aace-7a94dfb856a3.png)

**Comment by lerrryyy** [comment]
A very cool app!!! But I can't run it locally on my Visual Studio in any way, following all the instructions. For the Bunifu_UI_v1 library.5.3.dll I already had the Embedded Resource parameter and a lot of nuances that make everything more complicated. Could you record a video Could you record a video or somehow describe in more detail how you ran locally according to the instructions. It would help me a lot. Thank you in advance!
